tests/tpch/conftest.py

```diff
@@ -134,14 +135,17 @@ def spark(spark_setup, benchmark_time):
 
 def coiled_function(**kwargs):
     def _(function):
+        frame = sys._current_frames()[threading.get_ident()]
+        filename = frame.f_back.f_code.co_filename
+        module = filename.split("_")[-1].split(".")[0]  # dask, spark, ...
```

Comment:
Just a driveby comment but maybe `inspect.getmodule` or `function.__module__` is useful here

Reply:
Thanks!  That's much nicer.